### PR TITLE
Upload of bamhash files

### PR DESCRIPTION
**Pull request by AGoepfert**
Again, it would be great if someone correct toe tool dependency - everything else is working without any problems!

Thanks

### CODE REVIEW

**Comment by AGoepfert** [comment]
#229

**Comment by bgruening** [comment]
Awesome! Are you joining us today at the codefest?

File: tools/bamhash/bamhash.xml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                <param name="noQuality_fastq" type="boolean" checked="false" truevalue="--no-quality"
+                        falsevalue=""
+                        label="Ignore read quality?" 
+                        help="Do not use read quality as part of checksum"/>
+                <param name="noPaired_fastq" type="boolean" checked="false" truevalue="--no-paired"
+                        falsevalue=""
+                        label="List of fastq files are not paired-end reads?"/>
+            </when>
+        </conditional>
+        <param name="debug" type="boolean" checked="false" truevalue="--debug" falsevalue=""
```
Comment (bgruening) [inline tools/bamhash/bamhash.xml]: debug mode should not be offered to the user

Comment (AGoepfert) [inline tools/bamhash/bamhash.xml]: True.. sry! I forgot! 

I deleted them!

**Comment by bgruening** [comment]
I fixed a few things and uploaded everything to the TTS and TS.
Great work, thanks!

**Pull request merged**